### PR TITLE
fix everything ugh

### PR DESCRIPTION
**Pull request by rolyatmax**
it's all wrong

### CODE REVIEW

**Comment by rolyatmax** [comment]
@sricola RFR

**Comment by rolyatmax** [comment]
@mreiferson

**Comment by sricola** [comment]
I think @mccutchen may have some input as well.

File: index.html [hunk 1]
```diff
@@ -49,7 +48,7 @@
 <br />
 <br />
 My life time flights map - powered by [<a href="https://foursquare.com/sricola">foursquare</a>] and [<a href="http://www.jetlovers.com/profile/14768/" title="jetlovers">jetlovers</a>]<br />
-<div id="image_holer">
+<div class="image_holder">
 <img src="http://www.jetlovers.com/flights/map/?w=800&h=500&uid=14768" />
```
Comment (mccutchen) [inline index.html]: these ampersands need to be escaped

**Pull request merged**